editor.planx.uk/src/@planx/components/NextSteps/Public.tsx

```diff
@@ -1,25 +1,26 @@
-import Box from "@mui/material/Box";
-import Typography from "@mui/material/Typography";
 import { PublicProps } from "@planx/components/ui";
 import React from "react";
+import NextStepsList from "ui/NextStepsList";
 
 import Card from "../shared/Preview/Card";
 import QuestionHeader from "../shared/Preview/QuestionHeader";
 import { NextSteps } from "./model";
 
-type Props = PublicProps<NextSteps>;
+export type Props = PublicProps<NextSteps>;
 
-export default Component;
-
-function Component(props: Props) {
+const NextStepsComponent: React.FC<Props> = (props) => {
   return (
-    <Card handleSubmit={props.handleSubmit} isValid>
-      <QuestionHeader {...props} />
-      <Box>
-        <Typography variant="h2" style={{ color: "orangered" }}>
-          UNDER DEVELOPMENT
-        </Typography>
-      </Box>
+    <Card>
```

Comment:
to answer one of your questions - this line confirms you've set up the component correctly to not have a "Continue" button!

the props passed to the `<Card>` component determine how the "Continue" button is rendered:
- if `handleSubmit` is provided, then show the "Continue" button
- if `isValid` is falsey, then disable the button
- if both props are omitted like here, then no button at all

Reply:
Appreciate the clarification on this.